docs/operations-guide/running-the-metabase-jar-file.md

```diff
@@ -2,47 +2,134 @@
 
 To run Metabase via a JAR file, you will need to have a Java Runtime Environment (JRE) installed on your system.
 
-## Install Java JRE
+- [Quick start](#quick-start)
+- [Local installation](#local-installation)
+- [Production installation](#production-installation)
 
-We recommend the latest LTS version of JRE from [Eclipse Temurin](https://adoptium.net/) with HotSpot JVM and x64 architecture, but other [Java versions](./java-versions.md) are supported too.
+## Quick start
 
-## Download Metabase
+> The quick start is intended for running Metabase locally. See below for instructions on [running Metabase in production](#production-installation).
 
-Go to the [Metabase download page](https://metabase.com/start/jar.html) and download the latest release. 
+If you have Java installed:
 
-## Create a new directory and move the JAR into it
+1. [Download Metabase](https://metabase.com/start/jar.html).
+2. Create a new directory and move the Metabase JAR into it.
+3. Change into your new Metabase directory and run the JAR.
+   ```
+   java --jar metabase.jar
+   ```
 
-The JAR file will create some new files when it is run, so it's important to have a dedicated directory for it. Don't just leave it in your Downloads directory.
+Metabase will log its progress in the terminal as it starts up. Wait until you see "Metabase Initialization Complete" and visit [localhost:3000](http://localhost:3000/setup).
 
-## Launching Metabase
+## Local installation
+
+If you just want to try Metabase out, play around with Metabase, or just use Metabase on your local machine, Metabase ships with a default application database that you can use. **This setup is not meant for production**. If you intend to run Metabase for real at your organization, see [Production installation](#production-installation).
+
+The below instructions are the same as the quick start above, just with a little more context around each step.
+
+### 1. Install Java JRE
+
+You may already have Java installed. To check the version, open a terminal and run:
+
+```
+java --version
+```
+
+If Java isn't installed, you'll need to install it before you can run Metabase. We recommend the latest LTS version of JRE from [Eclipse Temurin](https://adoptium.net/) with HotSpot JVM and x64 architecture, but other [Java versions](./java-versions.md) are supported too.
```

Comment:
the architecture depends on the machine the user is running, so I'll remove the x64 arch

Reply:
This was added explicitly because some people were downloading 32bit Java on their 64bit system.